README_TO_RUN_WITH_DREMAPLACE

```diff
@@ -0,0 +1,7 @@
+1. Please follow the link and install/build DREAMPlace
+    https://github.com/limbo018/DREAMPlace
+2. Build OpenROAD following the README.md 
+3. In flow/Makefile update the below variables to run with DREAMPlace. (b) and (c) variables are used in "flow/scripts/prep_and_run_dream_place.sh"
+    a. DREAM_PLACE set to 1 to run global place using DREAMPlace.
+    b. PYTHON_CMD is the python path. Set it to DREAMPlace compatible python environment.
```

Comment:
This should be documented on the readthedocs.
https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/docs/user/FlowVariables.md

Reply:
Okay. I will update this file.

Reply:
Mention the OpenROAD PR here for reference when you create.

Reply:
I do not know how to change this (FlowVariables.md) file in a submodule and push the changes here?

Reply:
> Mention the OpenROAD PR here for reference when you create.

In which file should I mention the OR PR?